web/src/components/Incidents/IncidentsPage.jsx

```diff
@@ -315,7 +315,7 @@ const IncidentsPage = () => {
                   }
                 />
               </ToolbarItem>
-              <ToolbarItem className="pf-v5-m-align-right">
+              <ToolbarItem className="pf-m-align-right">
```

Comment:
The current version supported for the plugin is PF5. Is this a custom class or needs patternfly css?

Reply:
If you use className="pf-v5-m-align-right" this is the result -> it doesn't work
![image](https://github.com/user-attachments/assets/92b193e1-6a8d-44ee-84d5-826e00efff5b)

There is no classname like this in pf5, it's a mistake I had to fix to make the alignment work again.

Reply:
Then maybe we should use the component props instead of applying css: https://v5-archive.patternfly.org/components/toolbar#toolbaritem

Reply:
@jgbernalp I updated css with prop for toolbar item